appengine-java11/helidon-mp-gae-greet/src/main/gae/helidon-mp-app.yaml

```diff
@@ -0,0 +1,2 @@
+runtime: java11
```

Comment:
idiomatic location and name for this file is src/main/appengine/app.yaml

Why do we avoid idiomatic good defaults that are used in Cloud SDK (gcloud app deploy pom.xml) to deploy from source and all the Maven/Gradle plugins and IDE integratgions?

Reply:
We will be updating all Java 11 samples to use `gcloud app deploy pom.xml` shortly